recipes/nuraft/all/patches/0001-cmake_patches.patch

```diff
@@ -0,0 +1,262 @@
+ include_directories(BEFORE ${PROJECT_SOURCE_DIR}/examples)
+@@ -213,6 +231,7 @@ if (NOT (DISABLE_SSL GREATER 0))
+     )
+ endif ()
+ 
++include_directories(BEFORE "${PROJECT_SOURCE_DIR}/include/libnuraft")
+ # === Source files ===
+ set(RAFT_CORE
+     ${ROOT_SRC}/asio_service.cxx
+@@ -240,6 +259,8 @@ set(RAFT_CORE
+     ${ROOT_SRC}/snapshot_sync_req.cxx
+     ${ROOT_SRC}/srv_config.cxx
+     ${ROOT_SRC}/stat_mgr.cxx
++    examples/in_memory_log_store.cxx
+     )
+ add_library(RAFT_CORE_OBJ OBJECT ${RAFT_CORE})
++target_link_libraries(RAFT_CORE_OBJ Boost::Boost OpenSSL::OpenSSL)
+ 
+@@ -249,54 +269,26 @@ set(STATIC_LIB_SRC
+ # === Executables ===
+ set(LIBRARY_NAME "nuraft")
+ 
+-add_library(static_lib ${STATIC_LIB_SRC})
+-set_target_properties(static_lib PROPERTIES OUTPUT_NAME ${LIBRARY_NAME} CLEAN_DIRECT_OUTPUT 1)
+-
+-add_library(shared_lib SHARED ${STATIC_LIB_SRC})
+-set_target_properties(shared_lib PROPERTIES OUTPUT_NAME ${LIBRARY_NAME} CLEAN_DIRECT_OUTPUT 1)
+-if (APPLE)
+-    target_link_libraries(shared_lib ${LIBRARIES})
+-endif ()
+-
+-if (WIN32)
+-    set(LIBRARY_OUTPUT_NAME "${LIBRARY_NAME}.lib")
+-else ()
+-    set(LIBRARY_OUTPUT_NAME "lib${LIBRARY_NAME}.a")
+-endif ()
+-message(STATUS "Output library file name: ${LIBRARY_OUTPUT_NAME}")
+-
+-# === Examples ===
+-add_subdirectory("${PROJECT_SOURCE_DIR}/examples")
+-
++add_library(nuraft ${STATIC_LIB_SRC})
++target_link_libraries(nuraft Boost::Boost OpenSSL::OpenSSL)
++
++include_directories(BEFORE ${EXAMPLES_SRC})
++#set(TEST_SRCS
++#      "tests/src/test_async_result.cxx"
++#      "tests/src/test_buffer.cxx"
++#      "tests/src/test_everything_together.cxx"
++#      "tests/src/test_impls.cxx"
++#      "tests/src/test_logger.cxx"
++#      "tests/src/test_log_store.cxx"
++#      "tests/src/test_ptr.cxx"
++#      "tests/src/test_runner.cxx"
++#      "tests/src/test_scheduler.cxx"
++#      "tests/src/test_serialization.cxx"
++#      "tests/src/test_strfmt.cxx"
++#    )
++#add_executable(test_runner ${TEST_SRCS})
++#target_link_libraries(test_runner nuraft)
++#add_test(NAME TestRunner COMMAND test_runner)
+ 
+ # === Tests ===
+ add_subdirectory("${PROJECT_SOURCE_DIR}/tests")
+-
+-
+-if (CODE_COVERAGE GREATER 0)
+-    set(CODE_COVERAGE_DEPS
+-        raft_server_test
+-        failure_test
+-        asio_service_test
+-        buffer_test
+-        serialization_test
+-        timer_test
+-        strfmt_test
+-        stat_mgr_test
+-    )
+-
+-    # lcov
+-    SETUP_TARGET_FOR_COVERAGE(
+-        NAME raft_cov
+-        EXECUTABLE ./runtests.sh
+-        DEPENDENCIES ${CODE_COVERAGE_DEPS}
+-    )
+-endif()
+-
+-
+-# === Install Targets ===
+-install(TARGETS static_lib ARCHIVE DESTINATION lib)
+-install(TARGETS shared_lib LIBRARY DESTINATION lib)
+-install(DIRECTORY ${PROJECT_SOURCE_DIR}/include/libnuraft DESTINATION include)
+-
+-
+diff --git a/tests/CMakeLists.txt b/tests/CMakeLists.txt
+index 5ceccce..f6dc2f0 100644
+--- a/tests/CMakeLists.txt
++++ b/tests/CMakeLists.txt
```

Comment:
We do not run test https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#why-conancenter-does-not-build-and-execute-tests-in-recipes so this patch seems a little bit more agressive then is required

Can you please disable the tests from the build?